### DISCUSSION TITLE
Any example payload available?

### DISCUSSION THREAD

**Discussion by gjermshus** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#372] [Q&A] [answered]
Hi!
First, great project. I'm planning on buying the parts and implementing this for next season.
Until then, I started creating a new web interface that I wanted to use. 
I wonder if anyone has any examples of the payloads sent from the controller? I know I could compose them by looking at the server source code, but it would have been easier if there were examples of different payloads out there somewhere :)

Here are two screenshots of what I'm trying to create. This is built using React + TailwindCSS. I plan to recreate the current functionality and add some extra. For example a dashboard view for tablets.

![image](https://user-images.githubusercontent.com/7047589/188675723-d15e3fd6-393e-437a-b194-1b39c0f8875d.png)
![image](https://user-images.githubusercontent.com/7047589/188675784-df26cc09-3e02-4853-a262-68f26085b79d.png)

**Comment by visualapproach** [comment, marked as answer]
Hi, thx! You can see what the server sends by looking in the browser console.

**Reply by gjermshus** [reply]
Hi,
Thanks! I don't have the server available since I have not created it yet. I'm guessing that the software would need an actual nodeMCU to run on? Or does platform.io have an emulator?

**Reply by visualapproach** [reply]
Ah, I don't think so but if you search this forum there is several posts that has logs you can look at. I can't provide any myself because I'm out of town

**Reply by visualapproach** [reply]
Here is one example from a 4 wire pump #200

**Reply by gjermshus** [reply]
Thank you. I can create a mock server based on that!
